styles/styles.css

```diff
@@ -4,11 +4,22 @@ body {
 
 #answer_text {
     color: #ffffff;
-    font-size: min(24vw, 43vh);
+}
+
+@media (max-width: 100vh){
+    #answer_text {
+	font-size: 20vh;
+    }
+}
+
+@media (min-width: 100vh) {
+    #answer_text {
+	font-size: 25vh;

```

Comment:
Ville kanskje satt denne til noe som ikke bare utsetter problemet. Legge til `max-width: 95%;`, for eksempel? Da får vi linebreaks.